README.md

```diff
@@ -580,11 +583,12 @@ class XAxisExample extends React.PureComponent {
 
 | Property | Default | Description |
 | --- | --- | --- |
-| values | **required** | An array of values to render on the xAxis. Should preferably have the same length as the chart's dataPoints. |
+| data | **required** | An array of values or objects to render on the xAxis. Should preferably have the same length as the chart's dataPoints. If an object is used instead of a simple value, a `xAccessor`  prop might be needed to calculate the axis' extent. A data object can contain a `svg` property which allows you two override styles on that specific object  |
```

Comment:
The xAccessor is *required* when passing in complex objects.
"to" instead of "two"